src/components/home/partners/PrimaryPartner.jsx

```diff
@@ -58,12 +58,14 @@ const PrimaryPartner = () => {
         src={'/assets/TAOLogo.png'}
       />
       <Typography
-        variant="h4"
+        // variant="h4"
+        component="h4"
```

Comment:
I may be missing it, but are we skipping `h3`? I see the "building a better tomorrow..." as `h2` and then this.

Maybe that falls under accessibility check issue later...
because the titles in the grid below are also definitely headings of the paragraphs below them.

Reply:
I think I meant to set the variant to `h3` for that reason. I'll do that now.

We have an issue for overall code review anyway.

Reply:
but we need the component to change to `h3`... not sure what styling should be (variant), actually I think `h3` is used for other section headings too. If so, just `variant='h3'` then would make styling and component an `h3`... currently it's styled like `h3` but actually creating an `h4` on the DOM.